hathor/feature_activation/settings.py

```diff
@@ -0,0 +1,120 @@
+#  Copyright 2023 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from collections import defaultdict
+from typing import Any, NamedTuple, Optional
+
+from pydantic import Field, NonNegativeInt, PositiveInt, validator
+
+from hathor.feature_activation.feature import Feature
+from hathor.feature_activation.model.criteria import Criteria
+from hathor.utils.pydantic import BaseModel
+
+
+class Settings(BaseModel, validate_all=True):
+    """Feature Activation settings."""
+
+    # The number of blocks in the feature activation evaluation interval.
+    # Equivalent to 14 days (40320 * 30 seconds = 14 days)
+    evaluation_interval: PositiveInt = 40320
```

Comment:
Is it a default value for all features?

Reply:
No, this value is actually a constant per network, not a variable per feature. It has to be the same for all features. There's a lost thread about it here: https://github.com/HathorNetwork/rfcs/commit/1c874a535200492260e8b5deb5c837df61629c11#r112715956

Reply:
Got it. Thanks!